ddtrace/llmobs/_integrations/openai.py

```diff
@@ -247,3 +251,15 @@ def _extract_llmobs_metrics_tags(span: Span, resp: Any) -> Dict[str, Any]:
                 TOTAL_TOKENS_METRIC_KEY: prompt_tokens + completion_tokens,
             }
         return get_llmobs_metrics_tags("openai", span)
+
+    @classmethod
+    def is_default_base_url(cls, base_url: Optional[str] = None) -> bool:
+        if base_url is None:
+            return True
+        import re
+        from urllib.parse import urlparse
+
+        parsed_url = urlparse(base_url)
+        default_azure_endpoint_regex = re.compile("^[\\w.-]*openai\\.azure\\.com$")
```

Comment:
here as well, let's move imports to top of file and make regex strings constants